.github/workflows/commcare-android-pr-workflow.yml

```diff
@@ -0,0 +1,92 @@
+name: commcare-android PR CI
+
+on:
+  pull_request:
+    types:
+      - created
+  issue_comment:
```

Comment:
This is triggering a build on any comment we make on PR. We should remove it or make it trigger on a specific comment like 'retest this please' if possible.

Reply:
yes, there is a [PR](https://github.com/dimagi/commcare-android/pull/2644) to fix this already.